java/ql/src/Violations of Best Practice/Records/UselessMembersOfTheRecordsClass.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @id java/useless-member-of-the-record-class
+ * @name Useless serialization member of record class
+ * @description Using certain members of a record class during serialization will result in
+ *              those members being ignored.
+ * @previous-id java/useless-members-of-the-records-class
+ * @kind problem
+ * @precision very-high
+ * @problem.severity warning
+ * @tags quality
```

Comment:
Since this is a new query, the `ccr.qls` suite needs to be updated in the Autofix repo (otherwise the query will not get any autofix suggestions).

Reply:
Can we add the query to that .qls before a release happens?

Reply:
I believe so. I think that's what we've been doing.